azerothcore/azerothcore-wotlk
# Fix (DB/Creature): Azuremys Murloc Spawn Correction

## Changes Proposed:
-  Corrects the entire overpopulation of Murlocs in Azuremyst/Silvermyst Isle while also adding waypoints, data from sniffer.

## Issues Addressed:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/4136

## SOURCE:
Sniff Team

## Tests Performed:
- Build
- Performed


## How to Test the Changes:

1. Go to both areas of Azuremyst and Silvermyst Isle and you will see the over population .go c 17192, 17191, 17190
2. Make sure to check all the waypoints, all the go info is in the sql file itself also to make it easier to see/test each waypoint there is one waypoint that walks on the ground while in ocean, but this is a core issue as values have been checked.
3.
